### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by mwhooker**
automated tests to verify this works the way we expect it to

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from setuptools import setup, find_packages
+setup(
+    name="unbound-ec2",
+    version="0.1",
+    packages=find_packages(),
+    install_requires=[
+        'vaurien',
+        'dnspython'
```
Comment (whilp) [inline setup.py]: not used?

Comment (whilp) [inline setup.py]: oh lol, i see it now. ignore

File: tests/test_network.py [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+remote-control:
+        control-enable: no
+
+python:
+        python-script: "{conf.module}"
+"""
+    return tpl.format(conf=conf)
+
+class TestBadNetword(TestCase):
```
Comment (tomwans) [inline tests/test_network.py]: networddddd

**Comment by mwhooker** [comment]
I removed the hostnameless cert checking hack which can be out of scope for this PR.

I was able to resolve hosts successfully within the context of unbound, but wasn't able to test in VPC

**Comment by mwhooker** [comment]
looks like I can't comment in line...

there are a handful of simple specific strings in here. not sure it matters but eventually we might want to make it configurable. One thing we could do is automatically search for an instance that has the right zone in the tag

```
Values=*.{zone}
```

etc.

**Comment by tomwans** [comment]
@mwhooker, WDYT about just moving away from this process and going to a periodic DescribeInstances() poller, constructing a dict out-of-band, and feeding that data into unbound?

**Comment by mwhooker** [comment]
tom that is exactly the plan

File: invalidator.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        >>> rq.mark('foo')
+        >>> rq._age = lambda _: 2.0
+        >>> rq.pop()
+        'foo'
+        >>> "foo" in rq
+        False
+        """
+        
+        with self._history_lock:
+            _, name = max([(self.priority(v), k) for (k, v) in
```
Comment (swenson) [inline invalidator.py]: No need to instantiate the entire list twice:

``` python
max(((self.priority(v), k) for (k, v) in self.request_history.iteritems()))
```

Comment (mwhooker) [inline invalidator.py]: good call

File: unbound_ec2.py [hunk 4]
```diff
@@ -21,79 +21,105 @@
...
 from boto import config
 
-EC2_ENDPOINT = None
-EC2_ENDPOINT_ADDRESS = None
+
+
+import sched
+import heapq
+from collections import namedtuple
+class RequestQueue(object):
```
Comment (swenson) [inline unbound_ec2.py]: Is this still necessary?

Comment (mwhooker) [inline unbound_ec2.py]: this is all WIP. better disregard anything that doesn't make sense

**Pull request closed without merging**